Specs/Scene/Batched3DModel3DTileContentSpec.js

```diff
@@ -487,6 +488,30 @@ describe(
         class: metadataClass,
       });
 
+      const contentMetadataClass = new MetadataClass({
```

Comment:
Could we move these into a `beforeAll()` block like we've done before in past PRs?